engine/src/gfx/cockpit.cpp

```diff
@@ -396,7 +393,7 @@ float GameCockpit::LookupUnitStat(int stat, Unit *target) {
             float distance;
             const float locklight_time = configuration().graphics.locklight_time_flt;
             bool res = false;
-            if (tmpunit = target->Threat()) {
+            if ((tmpunit = target->Threat())) {
```

Comment:
As nice and elegant as this code is, I'd appreciate something along of:

```
tmpunit = target->Threat();
if(tmpunit) {
...
```

Reply:
Fair enough. CLion recommended that I put the assignment inside an extra pair of parentheses. But splitting it out onto its own line makes sense too.

Reply:
Done